0XXX-data-only-mobile-hotspots.md

```diff
@@ -0,0 +1,54 @@
+# HIP XXX: Data-only MOBILE Hotspots 
+
+- Author: @zer0tweets
+- Start Date: 
+- Category: Technical, Economic
+- Original HIP PR: [#]()
+- Tracking Issue: [#]()
+- Voting Requirements: veMOBILE Holders
+
+## Summary
+
+This Helium Improvement Proposal introduces the concept of Data-Only MOBILE Hotspots that  earn MOBILE for data transfer, but are not eligible to earn MOBILE for Proof-of-Coverage.
+
+## Motivation
+
+Most commercial venues like airports or malls already have Wi-Fi coverage. To grow coverage of the MOBILE network faster, it makes sense to develop an approach to make such pre-existing installations part of the MOBILE network. Wider coverage will result in Helium being more useful for telecom carriers looking to offload data into the network, which ultimately results in more utility for the network. 
+
+## Stakeholders
+
+Operators of the pre-existing Wi-Fi networks that support Passpoint will be able to add their Hotspots to the network, instead of having to buy brand new hardware from vendors certified by the Helium network 
+Network users will be able to connect to these Hotspots using the same authentication certificate they currently use for accessing greenfield Helium Hotspots 
```

Comment:
```suggestion
- Operators of the pre-existing Wi-Fi networks that support Passpoint will be able to add their Hotspots to the network, instead of having to buy brand new hardware from vendors certified by the Helium network 
- Network users will be able to connect to these Hotspots using the same authentication certificate they currently use for accessing greenfield Helium Hotspots 
```